source/companies.json

```
@@ -11,11 +11,26 @@
             "websiteUrl": "http://www.adobe.com/",
             "description": "Adobe provides digital media and digital marketing solutions."
         },
+        "admixer": {
+            "name": "Admixer",
+            "websiteUrl": "https://admixer.com/",
+            "description": "A supply-side platform that connects publishers with global advertisers and agencies, supporting all ad formats and offering real-time revenue optimization."
+        },
+        "adprofex": {
+            "name": "AdProfex",
+            "websiteUrl": "https://adprofex.com/",
+            "description": "AdProfex is a new type of advertising network for news websites and blogs that makes generating revenue a snap."
+        },
         "advance": {
             "name": "Advance Publications, Inc.",
             "websiteUrl": "https://www.advance.com/",
             "description": "Advance Publications is a holding company whose media properties include newspapers, Conde Nast Publications (Vogue, Vanity Fair, GQ, Architectural Digest and other magazines), Parade Publications, Fairchild Publications and American City Business Journals."
         },
+        "affilbox": {
+            "name": "Affilbox",
+            "websiteUrl": "https://affilbox.com/",
+            "description": "AffilBox is a affiliate software."
```

```suggestion
            "description": "AffilBox is an affiliate software."
```

Please ignore if I'm wrong.